src/RadixDlt.NetworkGateway.GatewayApi/Exceptions/KnownGatewayErrorException.cs

```diff
@@ -72,13 +72,13 @@ public abstract class KnownGatewayErrorException : Exception
 {
     public int StatusCode { get; }
 
-    public GatewayModel.GatewayError GatewayError { get; }
+    public GatewayModel.GatewayError? GatewayError { get; }
```

Comment:
Do we want to add GatewayErrors for:

- ClientClosedConnection
- BadRequestException

Reply:
I don't think we should document such low-level / technical exceptional situations like closed connection. There's a non-negligible chance such closed connection might be short-circuited by software higher up in the overall infrastructure pipeline.

Reply:
ok so lets keep that as nullable.